tests/compare-compile-time.fmf

```diff
@@ -0,0 +1,96 @@
+# This test plan compares three versions of LLVM and their compile time:
+#
+#   1. The PGOed LLVM snapshot living in the daily copr repo:
+#      llvm-snapshots-pgo-$YYYYMMDD
+#   2. The regular LLVM snapshot also living in another daily copr repo:
+#      llvm-snapshots-big-merge-$YYYYMMDD
+#   3. The system LLVM living in the regular fedora repository.
+#
+# NOTE: The comparison happens on the same machine for each chroot to be fair.
+#
+# We store all comparison results in the test plans artifact directory (see
+# `TMT_PLAN_DATA` in
+# https://tmt.readthedocs.io/en/stable/overview.html#step-variables). Expect
+# these files for each passing plan.
+#
+#   1. pgo.json
+#   2. snapshot.json
+#   3. system.json
+#   4. pgo_vs_snapshot.compile_time.txt
+#   5. pgo_vs_system.compile_time.txt
+#   6. snapshot_vs_system.compile_time.txt
+#   7. result.csv
+#
+# In order to run this test plan on testing-farm you need to make a request like
+# so:
+#
+#   $ testing-farm request \
+#     --compose Fedora-Rawhide \
+#     --git-url https://github.com/kwk/llvm-snapshots \
+#     --git-ref compile-time-perf \
```

Comment:
I think this is misleading. This would tell testing farm to checkout to branch named `compile-time-perf` from your (@kwk) fork repo. While this is completely ok right now, this can be confusing for future users who read this. For them, It might make more sense to use `--git-url https://github.com/fedora-llvm-team/llvm-snapshots --git-ref main`

Reply:
You're right. I need to change this. In order to get the command right every time I run this I've made the command part of the documentation. I've changed it back to what will finally be more correct in e36a7fecab8b23ed09ed919bf6d18cac3f17fb05.